cloud_governance/common/utils/configs.py

```diff
@@ -21,3 +21,31 @@
 EC2_NAMESPACE = 'AWS/EC2'
 CLOUDWATCH_METRICS_AVAILABLE_DAYS = 14
 AWS_DEFAULT_GLOBAL_REGION = 'us-east-1'
+
+# X86 to Graviton
+GRAVITON_MAPPINGS = {
+    'c6a': 'c6g',
+    'c6i': 'c6g',
+    'c6in': 'c6gn',
+    'c7': 'c7g',
+    'c7i': 'c7g',
+    'c7a': 'c7g',
+    'c7i-flex': 'c7g',
+    'g5': 'g5g',
+    'hpc7a': 'hpc7g',
+    'i4i': 'i4g',
+    'm5': 'm6g',
+    'm5a': 'm6g',
+    'm5n': 'm6g',
+    'm5zn': 'm6g',
+    'm6i': 'm6g',
+    'm6in': 'm6g',
+    'm6a': 'm6g',
+    'm7i': 'm7g',
+    'm7a': 'm7g',
+    'm7i-flex': 'm7g',
+    'r7i': 'r8g',
+    'r7a': 'r7g',
+    'r6i': 'r6g',
+    't3': 't4g'
+}
```

Comment:
can we fetch it from AWS automatically and not use it manually ?

Reply:
It's mapping intel to the graviton. We can't fetch dynamically because we don't know which graviton instance to fetch.

Reply:
Its good for now, in the future we must build it dynamically

Reply:
Done